src/Evolution/Executables/GeneralizedHarmonic/CMakeLists.txt

```diff
@@ -42,3 +42,14 @@ target_link_libraries(
   PRIVATE
   GeneralRelativitySolutions
   )
+
+add_generalized_harmonic_executable(
+  KerrSchildNumericInitialData
```

Comment:
I'm not sure KerrSchildNumericInitialData is a good name, because it implies the numeric initial data is for KerrSchild. What if it's for perturbed KerrSchild, or for a binary? Maybe the name should just be EvolveGeneraliedHarmonicNumericInitialData? As long as the outer boundary is far enough away, the KerrSchild BC is still ok, at least for a little while (until GWs reach the outer boundary)

Reply:
Right, it's the GH executable with numeric initial data and KerrSchild boundary conditions. I'm happy for you and @nilsdeppe to choose a name for it :)